test/OptimisticArbitrator/OptimisticArbitrator.Lifecycle.ts

```diff
@@ -0,0 +1,123 @@
+import {
+  ExpandedERC20Ethers,
+  MockOracleAncillaryEthers,
+  OptimisticOracleV2Ethers,
+  StoreEthers,
+} from "@uma/contracts-node";
+import { OptimisticArbitrator } from "../../typechain";
+import { optimisticArbitratorFixture, seedAndApprove, umaEcosystemFixture } from "../fixtures";
+import { ethers, expect, SignerWithAddress } from "../utils";
+import hre from "hardhat";
+
+let optimisticArbitrator: OptimisticArbitrator, usdc: ExpandedERC20Ethers;
+let optimisticOracle: OptimisticOracleV2Ethers, store: StoreEthers, mockOracle: MockOracleAncillaryEthers;
+let requester: SignerWithAddress, proposer: SignerWithAddress, disputer: SignerWithAddress;
+
+describe("OptimisticArbitrator: Lifecycle", function () {
+  beforeEach(async function () {
+    // Load accounts and run fixtures to set up tests.
+    [requester, proposer, disputer] = await ethers.getSigners();
```

Comment:
`requester` overlaps with contract deployer, so better to skip 1 index position.

Also, `proposer` and `disputer` do not get used in these tests as all transactions are performed from default account.